.gitignore

```
@@ -0,0 +1,2 @@
+.flatpak-builder
```

These would be useful if if you want to build locally for testing purpose.
```suggestion
.flatpak-builder
builddir
repo
*.flatpak
```